.circleci/config.yml

```diff
@@ -0,0 +1,125 @@
+version: 2.1
+workflows:
+    lint_and_test:
+        jobs:
```

Comment:
Can we add branch filter? I think we only need run ci for develop and master.

Reply:
We can add branch filters but I think it's reasonable to test all pushes. That's what we do now, I think we should at least test pull request branches. With circleci we get 10x more free execution time on machines that are 2x as big as what github gives us, might as well use it 😀